README.md

```diff
@@ -1 +1,62 @@
-# dynamodb-python
\ No newline at end of file
+# dynamodb-python
+
+## Installation
+
+## Requirements
+
+### Dependencies
+
+- python = "^3.9"
+- ddbcereal
+- botocore
+- boto3
+- requests
+
+### Credentials
+
+You will first need to set your AWS credentials. Since this library uses `boto3` under the hood, you can use the same methods as described in the [boto3 documentation](https://boto3.readthedocs.io/en/latest/guide/configuration.html). In short, AWS looks for credentials in these places:
+
+1. Environment variables
+2. Shared credentials file (`~/.aws/credentials`)
+3. AWS config file (`~/.aws/config`)
+
+You cacn also pass a dictionary to `DynamoDB` class:
+
+```python
+from dynamodb_python import DynamoDB
+
+dynamodb = DynamoDB(credentials={
+    "aws_access_key_id": ACCESS_KEY,
```

Comment:
But DynamoDB can automatically infer those creds, no?

Reply:
It can, but I wanted to add an option to pass creds manually.